### ISSUE TITLE
Could Output message include name of calendar?

### ISSUE THREAD

**Issue by ukmoose** [naimo84/node-red-contrib-ical-events#54]
**Is your feature request related to a problem? Please describe.**
I'm trying to poll multiple calendars to create a printable weekly family diary.  

My flow currently has to poll in parallel to avoid the issue caused by #53 
but the object from each calendar event doesn't appear to include the name of the calendar, so I need to add that to the event object before concatenating the outputs.

**Describe the solution you'd like**
It would be great if the object for each calendar event included the name of the calendar it comes from

**Describe alternatives you've considered**
Currently I iterate through the output array to add the data into the event object, but it would be great if it was already available!

**Additional context**
n/a

**Comment by naimo84** [comment]
Hi ukmoose,

unfortunatelly it's not possible to get the calendarName from the iCloud itself, because it's not part of the ics response.
But, what I can offer you is to output the "Config-Name" as calenderName?

So, this one as calendarName?

![image](https://user-images.githubusercontent.com/11535502/86472857-21bfdc80-bd40-11ea-8f38-b1e7cf8b3d53.png)


Greets,
Benjamin

**Comment by ukmoose** [comment]
Sound great!  Just anyway to identify the source of the event.


Many thanks




On 3 Jul 2020, at 14:16, Benjamin <notifications@github.com> wrote:


Hi ukmoose,

unfortunatelly it's not possible to get the calendarName from the iCloud itself, because it's not part of the ics response.
But, what I can offer you is to output the "Config-Name" as calenderName?

So, this one as calendarName?

 <https://user-images.githubusercontent.com/11535502/86472857-21bfdc80-bd40-11ea-8f38-b1e7cf8b3d53.png>
Greets,
Benjamin

**Comment by naimo84** [comment]
v0.16.0 is on the way 😉
